gitpod-io/gitpod
# [Self-Hosted] Allow integrating with 'github.com' without a GitHub App

## Description


Allow integrating with 'github.com' without requiring a GitHub App. This simply reuses the same common code as GitLab, Bitbucket and GitHub Enterprise, i.e. API and webhooks.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/9163

## How to test


1. Don't follow any of the steps from https://www.notion.so/gitpod/How-to-deploy-a-PR-with-a-working-GitHub-App-integration-d297a1ef2f7b4b3aa8483b2ae9b47da2
2. You should still be able to add Projects and run Prebuilds for `github.com` repos 🎉

## Release Notes

```release-note
Allow integrating with 'github.com' without a GitHub App
```

## Documentation
